T/Tar/build_tarballs.jl

```diff
@@ -3,19 +3,20 @@
 using BinaryBuilder
 
 name = "Tar"
-version = v"1.34"
+version = v"1.35"
 
 # Collection of sources required to build tar
 sources = [
     ArchiveSource("https://ftp.gnu.org/gnu/tar/tar-$(version.major).$(version.minor).tar.xz",
-                  "63bebd26879c5e1eea4352f0d03c991f966aeb3ddeb3c7445c902568d5411d28"),
+                  "4d62ff37342ec7aed748535323930c7cf94acf71c3591882b26a7ea50f3edc16"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/tar-*/
 export FORCE_UNSAFE_CONFIGURE=1
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+# We disable the year 2038 check because we don't have an alternative on the affected systems
```

Comment:
Maybe do this only for 32-bit platforms though?

Reply:
Done.